node/README.md

```diff
@@ -0,0 +1,98 @@
+## Joystream-Node - full node/validator
+
+![ Nodes for Joystream](./validator-node-banner.svg)
+
+joystream-node is the main server application that connects to the network, synchronizes the blockchain with other nodes and produces blocks if configured as a validator node.
+
+To setup a full node and validator review the [advanced guide from the helpdesk](https://github.com/Joystream/helpdesk/tree/master/roles/validators).
+
+### Pre-built Binaries
+
+The latest pre-built binaries can be downloaded from the [releases](https://github.com/Joystream/joystream/releases) page.
+Generally these will be built from master branch and relevant to the current active testnet.
+
+### Building from source
+
+Clone the repository and install build tools:
+
+```bash
+git clone https://github.com/Joystream/joystream.git
+
+cd joystream/
+
+./setup.sh
+```
+
+compile the node and runtime
+
+```bash
+cargo build --release
+```
+
+This produces the binary in `./target/release/joystream-node`
+
+### Running Local development chain
+
+```bash
+cargo run --release -- --dev
+```
+
+If you repeatedly need to restart a new chain,
+This script will build and run a fresh new local development chain purging existing chain data:
+
+```bash
+./scripts/run-dev-chain.sh
+```
+
+### Joystream public testnet
+
+Use the `--chain` argument, and specify the path to the genesis chain.json file for that public network. The joystream public networks are found in [../testnets/](../testnets/)
```

Comment:
```suggestion
Use the `--chain` argument, and specify the path to the genesis `chain.json` file for that public network. The JSON "chain spec" files for Joystream public networks can be found in [../testnets/](../testnets/).
```
Also noticed that there only seems to be a `rome.json` in that folder, does it need updating?

Reply:
If you remember, rome was a new chain, and then the later releases were runtime upgrades and that doesn't require a new chain (genesis) file. I don't think we need to add any other staging or testing networks.